### PR TITLE
Use the daemonset used in GKE documentation in the test.

### PR DESCRIPTION
**Pull request by mindprince**
This depends on https://github.com/kubernetes/kubernetes/pull/58782

/cc @vishh @jiayingz

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -7892,6 +7892,7 @@
     "args": [
       "--check-leaked-resources",
       "--deployment=gke",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gke-staging-latest-device-plugin-gpu.env",
```
Comment (krzyzacy) [inline jobs/config.json]: just add `--env=NVIDIA_DRIVER_INSTALLER_DAEMONSET=https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/k8s-1.9/nvidia-driver-installer/cos/daemonset-preloaded.yaml`

Comment (mindprince) [reply jobs/config.json]: Done. Thanks!

**Comment by krzyzacy** [comment]
hummmm something breaks the presubmit :-( @BenTheElder 🔥

**Pull request merged**